### DISCUSSION TITLE
New User Notifications

### DISCUSSION THREAD

**Discussion by fast4x** [fast4x/RiMusic#2925] [Ideas]
Could use 
- Toast, simple
- Snackbar, modern

which do you prefer?

**Comment by lebgdeyoutube** [comment]
Are the simple toasts currently used?

**Reply by fast4x** [reply]
No, actually is a custom toast called SmartToast

**Comment by aneesh1122** [comment]
This is snackbar

![Screenshot_2024-07-26-00-26-35-981_com.brave.browser.png](https://github.com/user-attachments/assets/4702eba8-64fb-42d1-8b0c-ec4f7f3acd87)

**Comment by lebgdeyoutube** [comment]
And that’s what simple toast is? 
![toast](https://github.com/user-attachments/assets/27df56e3-2f0f-4ee2-974c-c6deb2900a9e)

**Comment by ALMIGHTY-VC** [comment]
How bout toast as an essential option and snackbar as an modern option like the current ones

**Reply by fast4x** [reply]
It seems a good idea

**Reply by ALMIGHTY-VC** [reply]
Hmm yes it just follows the apps aesthetics and my mind was like why not both

**Reply by fast4x** [reply]
Snackbar under development

**Reply by aneesh1122** [reply]
> Snackbar under development 

Yaay

**Reply by ALMIGHTY-VC** [reply]
> Snackbar under development

Noice
